eacl/target.go

```diff
@@ -58,6 +60,26 @@ func (t *Target) SetBinaryKeys(keys [][]byte) {
 	t.keys = keys
 }
 
+// Accounts returns list of accounts to identify target subject in a binary format.
+//
+// Each element of the resulting slice is a serialized account from the public key.
```

Comment:
> from the public key

again, not always true - multisigs exist

```suggestion
// Each element of the resulting slice is a 20 byte account script hash.
```

and then add `Instead of a script hash there can also be a 33 byte public key binary.`